apps/prairielearn/src/components/QuestionsTable.html.ts

```diff
@@ -239,12 +239,29 @@ export function QuestionsTable({
                   >question documentation</a
                 >.
               </p>
-              <form class="ml-1 btn-group" method="POST">
-                <button class="btn btn-sm btn-primary">
-                  <i class="fa fa-plus" aria-hidden="true"></i>
-                  <span>Add question</span>
-                </button>
-              </form>
+              ${showAddQuestionButton
+                ? html`
+                    <div class="d-flex flex-row flex-wrap justify-content-center gap-3">
+                      <form class="ml-1 btn-group" method="POST">
+                        <button class="btn btn-sm btn-primary">
+                          <i class="fa fa-plus" aria-hidden="true"></i>
+                          Add question
+                        </button>
+                      </form>
+                      ${showAiGenerateQuestionButton
+                        ? html`
+                            <a
+                              class="btn btn-sm btn-primary"
+                              href="${urlPrefix}/ai_generate_question_drafts"
```

Comment:
While I think this is completely fine for now, at some point it might be nice if we can jump straight into the Ai question creation flow from here.

Reply:
We'll do that alongside item 6 of https://github.com/PrairieLearn/PrairieLearn/issues/11194.

Reply:
Perfect!